### PR TITLE
Adds gives and gets for Pyrsia incubation

### PR DESCRIPTION
**Pull request by betarelease**
These gives and gets are a starting point for a discussion and should be amended once discussed.

### CODE REVIEW

File: process/README.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+- Meetup Member List
+- CLA transfer to Linux Foundation
+- Reporting required by TAC and Governing Board
+
+#### Gets - Benefits
+
+- Additional governance by OpenSSF Governing Board
+- Oversight by OpenSSF TAC
+- Access to OpenSSF Ambassador program
+- Access to OpenSSF Marketing
```
Comment (sbtaylor15) [inline process/README.md:89]: Marketing would provide Webinars, Youtube Live, Twitch, Linkedin Live, and Podcasts

File: process/README.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+- CLA transfer to Linux Foundation
+- Reporting required by TAC and Governing Board
+- Security Audit by 3rd Party to Graduate
+
+#### Gets - Benefits
+
+- Additional governance by OpenSSF Governing Board
+- Oversight by OpenSSF TAC
+- Access to OpenSSF Ambassador program
+- Access to OpenSSF Marketing
```
Comment (sbtaylor15) [inline process/README.md:146]: Marketing would provide Webinars, Youtube Live, Twitch, Linkedin Live, and Podcasts

File: process/README.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+- Meetup Member List
+- CLA transfer to Linux Foundation
+- Reporting required by TAC and Governing Board
+
+#### Gets - Benefits
+
+- Additional governance by OpenSSF Governing Board
+- Oversight by OpenSSF TAC
+- Access to OpenSSF Ambassador program
+- Access to OpenSSF Marketing
```
Comment (sbtaylor15) [inline process/README.md:209]: Marketing would provide Webinars, Youtube Live, Twitch, Linkedin Live, and Podcasts

**Comment by lehors** [comment]
I think someone with the right privileges needs to close this.

**Comment by lehors** [comment]
Pyrsia moved to CDF and this is no longer relevant.

**Pull request closed without merging**